### DISCUSSION TITLE
Does Github provide information on their RTO and RPO?

### DISCUSSION THREAD

**Discussion by blurredminds** [community/community#62423] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Our client needs to know what the Recovery Time Objective (RTO) and Recovery Point Objective (RPO) are for Github. Does Github provide that sort of information?

**Comment by ajonesCRWD** [comment]
I am also looking for this information

**Comment by vigneshsundaram** [comment]
I eagerly praying for reply
